internal/template/funcmap.go

```diff
@@ -139,3 +140,8 @@ func regexReplaceAll(regex string, s string, repl string) (string, error) {
 	}
 	return r.ReplaceAllString(s, repl), nil
 }
+
+// makeSlice combines the arguments into an array and returns the array.
+func makeSlice(args ...string) []string {
```

Comment:
we can probably simply name it `list` and extend it to be interface{}

```
func list(v ...interface{}) []interface{} {
	return v
}
```

Reply:
+1

Reply:
Done